src/docker_snapshot/container.py

```diff
@@ -10,15 +11,27 @@
 from docker_snapshot import images, settings
 
 
+if t.TYPE_CHECKING:
+    from types import TracebackType
+    from docker.client import DockerClient
+    from docker.models.containers import Container
+    from docker.models.images import Image
+    from docker.models.volumes import Volume
+    from typing_extensions import ParamSpec
+
+    P = ParamSpec("P")
+    R = t.TypeVar("R")
```

Comment:
I'm sorry, what? :D

Reply:
...any specific question? 😄

Reply:
They're used to annotate the `requires_helper_container` decorator below, for reference.

Reply:
![image](https://github.com/user-attachments/assets/c8bab4e3-699f-4415-95e0-db5f2fc706e7)
PyCharm is showing this, is this just bad support for typing or something wrong with the code itself?

Reply:
Myeah, looks like the string syntax might be unsupported by some part of the 3.9 tooling, but not entirely sure what it is. Mypy is fine with this at least...